invenio_base/app.py

```diff
@@ -194,7 +194,7 @@ def blueprint_loader(app, entry_points=None, modules=None):
 
 
 def converter_loader(app, entry_points=None, modules=None):
-    """Default converter loader.
+    """Return default converter loader.
```

Comment:
True, pydocstyle 2.0.0 wants the imperative verb in the beginning, create or define?